modules/light-clients/08-wasm/types/client_state.go

```diff
@@ -86,9 +86,14 @@ func (cs ClientState) GetTimestampAtHeight(
 	cdc codec.BinaryCodec,
 	height exported.Height,
 ) (uint64, error) {
+	timestampHeight, ok := height.(clienttypes.Height)
+	if !ok {
+		return 0, errorsmod.Wrapf(ibcerrors.ErrInvalidType, "expected %T, got %T", clienttypes.Height{}, height)
```

Comment:
we mostly use the approach used here. We should open a larger issue about fixing the way we do this tho since I see we're not consistent all over the place, not just 08-wasm.

Reply:
Relayers provide proof heights as values and not as pointers https://github.com/cosmos/ibc-go/blob/main/proto/ibc/core/channel/v1/tx.proto#L185